src/components/modal/v2/parts/BodyContent.jsx

```diff
@@ -82,7 +83,9 @@ const BodyContent = () => {
     // Add views to viewComponents object where the keys are the product name and the values are the view component
     const viewComponents = {
         [VIEW_IDS.PAYPAL_CREDIT_NO_INTEREST]: <NoInterest content={content} openProductList={openProductList} />,
-        [VIEW_IDS.PAY_LATER_LONG_TERM]: <LongTerm content={content} openProductList={openProductList} />,
+        [VIEW_IDS.PAY_LATER_LONG_TERM]: (
+            <LongTerm content={content} useV4Design={useV4Design} openProductList={openProductList} />
```

Comment:
```suggestion
            <LongTerm content={content} productMeta={productMeta} openProductList={openProductList} />
```
We can pass in `productMeta` instead as a prop since the `useV4Design` will be in there.